mesonbuild/modules/unstable_wayland.py

```diff
@@ -12,24 +12,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
 import os
+import typing as T
+
+from mesonbuild.programs import OverrideProgram
```

Comment:
Shouldn't this be grouped with the other imports from mesonbuild? And why does this use absolute imports, while all the others are relative ones?

Reply:
Also this isn't used at runtime, only as an annotation.